projects/plugins/jetpack/modules/masterbar/admin-menu/class-wpcom-admin-menu.php

```diff
@@ -45,6 +45,8 @@ public function reregister_menu_items() {
 			$this->add_new_site_link();
 		}
 
+		$this->add_beta_testing_menu();
```

Comment:
Do we have plans to support the Beta Testing Page on Atomic sites as well? If that's the case, we should add this to `class-atomic-admin-menu.php` too.

Reply:
Thanks @mmtr, that's a good point and we've decided to go ahead and add this link to Atomic as well.

For now, I'd say we keep this PR as is, non-Atomic and I'll open a follow-up diff + PR which to enable Atomic, and will remove the link from P2 (as per 3513-gh-Automattic/p2).